src/main/kotlin/org/stellar/walletsdk/Auth.kt

```diff
@@ -11,6 +11,21 @@ import org.stellar.sdk.Transaction
 import org.stellar.walletsdk.util.GsonUtils
 import org.stellar.walletsdk.util.OkHttpUtils
 
+/**
+ * Authenticate to an external server using
+ * [SEP-10](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md).
+ *
+ * @property accountAddress Stellar address of the account authenticating
+ * @property authEndpoint Authentication endpoint URL
```

Comment:
It might be helpful to call this param `webAuthEndpoint` to align with the naming convention used in the SEP's so users who are learning can recognize what this maps to

Here's a good article that Leigh shared with me: https://martinfowler.com/bliki/UbiquitousLanguage.html